addon/-private/system/snapshot.js

```diff
@@ -65,7 +65,12 @@ export default class Snapshot {
      */
     this.modelName = internalModel.modelName;
 
-    this._changedAttributes = internalModel.changedAttributes();
+    if (internalModel.isLoading()) {
```

Comment:
I am skeptical. What about `reload` ?

Reply:
@runspired what's your preferred branch?  the state here is `root.loading`